src/ert/gui/ertwidgets/multilinestringbox.py

```diff
@@ -0,0 +1,112 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Any, Optional
+
+from qtpy.QtGui import QPalette
+from qtpy.QtWidgets import QTextEdit
+
+from .validationsupport import ValidationSupport
+
+if TYPE_CHECKING:
+    from ert.validation import ArgumentDefinition
+
+    from .models import TextModel
+
+
+class MultiLineStringBox(QTextEdit):
```

Comment:
This new class is a copy of StringBox, but it inherits from QTextEdit rather than QLineEdit.